### PR TITLE
LPS-200197: Add test for Import Site Scope Object Entry

### PR DESCRIPTION
**Pull request by sergiojimcos**
**What is new?**
- Add test case for Import Site Scope Object Entry

**JIRA Related Ticket**
https://liferay.atlassian.net/browse/LPS-200197

### CODE REVIEW

**Comment by sergiojimcos** [comment]
ci:test:relevant

File: modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java [hunk 1]
```diff
@@ -557,6 +557,25 @@ public void testImportObjectDefinition() throws Exception {
...
+		_objectDefinition1 = _publishObjectDefinition(
+			TestPropsValues.getCompanyId(), "TestObject",
+			ObjectDefinitionConstants.SCOPE_SITE, TestPropsValues.getUser());
+
+		// default group
+
+		_testImportSiteScopeObjectEntry(
+			TestPropsValues.getGroupId(), _OBJECT_ENTRY_ERC_1);
+
+		Group group = GroupTestUtil.addGroup();
```
Comment (4lejandrito) [inline modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java:571]: Is this deleted after the test runs?

I think so, but... can you double check?

File: modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java [hunk 2]
```diff
@@ -557,6 +557,25 @@ public void testImportObjectDefinition() throws Exception {
 			_objectDefinitionImportFieldNames, jsonNode.get(0));
 	}
 
+	@Test
+	public void testImportSiteScopeObjectEntry() throws Exception {
```
[the remark this answers is not in the document]
Comment (4lejandrito) [reply modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java:561]: ok but don't forget about it.

**Comment by sergiojimcos** [comment]
ci:test:sf

**Comment by sergiojimcos** [comment]
ci:test:relevant

File: modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java (hunk 1 above)
Comment (4lejandrito) [reply modules/apps/batch-planner/batch-planner-test/src/testIntegration/java/com/liferay/batch/planner/batch/engine/broker/test/BatchEngineBrokerTest.java:571]: Create the group in the setup and annotate it with @DeleteAfterTestRun.

**Comment by sergiojimcos** [comment]
ci:test:sf

**Comment by sergiojimcos** [comment]
ci:test:relevant

**Review by 4lejandrito — APPROVED** [review]

**Pull request closed without merging**